server/Entities/UserRepository.cs

```diff
@@ -22,5 +22,13 @@ public void CreateUser(User user)
             RepositoryContext.Users.Add(user);
             RepositoryContext.SaveChanges();
         }
+
+        public void UpdateUserPassword(User user)
```

Comment:
it seems strange, i expect here not mutated user (user in db exists with old password yet) and password separately.
Actually, one more moment, usually if some entity exists in name of function usually programmers expects that the need to pass this entity
so UpdateUserPassword(User user, string password)

Reply:
okey